08-transport.Rmd

```diff
@@ -523,13 +587,14 @@ plot(ways_sln@sl$geometry, add = TRUE)
 
 Chapter \@ref(location) concluded the case study by identifying locations for creating new bike shops.
 Similarly this chapter's final practical section will identify locations for construction of new transport infrastructure.
-After researching Bristol's transport system at multiple levels we can now tie-together various the strands covered in sections \@ref(transport-zones) to \@ref(route-networks).
+After researching Bristol's transport system at multiple levels we can now tie-together the various strands covered in sections \@ref(transport-zones) to \@ref(route-networks).
 There is another between the chapters:
 if the new infrastructure succeeds in getting people walking and cycling perhaps sustainable transport companies --- like the chain of German cycling shops described in Chapter \@ref(location) --- will be encouraged to expand internationally and set-up in Bristol!
 
-In fact we have already done all the hard work needed to priorities cycling infrastructure:
-we have short but car-dependent commuting routes (generated from desire lines) created in section \@ref(routes), desire lines representing the first leg to public transport nodes created in section \@ref(nodes) and the ability to analyse route networks in terms of 'betweenness' and other conceptions for graph theory, covered in the previous section.
-Rather than describe how each of these elements will be combined the final code chunk should speak for itself:
+In fact we have already done all the hard work needed where to prioritize cycling infrastructure:
+we have short but car-dependent commuting routes (generated from desire lines) created in section \@ref(routes), desire lines representing the first leg to public transport nodes created in section \@ref(nodes) and the ability to analyze route networks in terms of 'betweenness' and other conceptions for graph theory, covered in the previous section.
+<!-- maybe it would be better to explain the code a little bit-->
```

Comment:
I think the code needs some explanation.

Reply:
Explained now - agreed I got lazy!

Reply:
😄  but I perfectly understand!